v2/internal/frontend/runtime/desktop/events.js

```diff
@@ -173,4 +179,21 @@ export function EventsOff(eventName, ...additionalEventNames) {
             removeListener(eventName)
         })
     }
-}
\ No newline at end of file
+}
+
+/**
+ * ListenerOff unregisters a listener previously registered with EventsOn
+ *
+ * @param {Listener} listener
+ */
+ function ListenerOff(listener) {
+    // Remove local listener
+    eventListeners[eventName] = eventListeners[eventName].filter(l => l !== listener);
```

Comment:
Should this be `eventListeners[listener.eventName]` ?

Reply:
Yup, it should be. Btw, how do you test these changes?